specification/paths/CreateManifestForCollection.json

```diff
@@ -6,7 +6,8 @@
     "security": [
       {
         "OAuth2": [
-          "shipments.manage"
+          "collections.view",
```

Comment:
Lets also update the `get-collections-by-id` endpoint 
<img width="620" height="252" alt="image" src="https://github.com/user-attachments/assets/f9587e59-dc77-47b7-9164-8e9c173dfa3c" />